nixos/modules/services/misc/snapper.nix

```diff
@@ -23,6 +23,8 @@ let
       "yes"
     else if v == false then
       "no"
+    else if isInt v then
+      "\"${v}\""
```

Comment:
This will fail - you'll have to explicitly use `toString` here

Reply:
Also, the `isString` check below seems redundant

Reply:
Fixed

Reply:
The fix also seems redundant :) `toJSON` will already convert it to a string.

Reply:
That was the legacy code here, no intention for me.

Reply:
Change it !

Reply:
I'm referring to the code you added in this PR - `toString v` is the same as `toJSON v`, for some int `v`

Reply:
Yes sorry, I'm dumb, and answered to quickly; sorry about that.